packages/open-truss/bin/open-truss

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
When npm packages are installed they add any binaries to the path (global path if installed globally). I felt that name spacing out binaries in  `open-truss` would reduce the likelihood of collisions with other binaries, especially since I'm not sure if any errors are thrown when there are collisions and instead binaries paths may silently get overridden.